orca-pipelinetemplate/src/main/java/com/netflix/spinnaker/orca/config/PipelineTemplateConfiguration.java

```diff
@@ -16,11 +16,24 @@
 
 package com.netflix.spinnaker.orca.config;
 
+import com.github.jknack.handlebars.Handlebars;
 import com.netflix.spinnaker.orca.pipelinetemplate.PipelineTemplateModule;
+import com.netflix.spinnaker.orca.pipelinetemplate.PipelineTemplateObjectMapper;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
 
 @ConditionalOnProperty("pipelineTemplate.enabled")
 @ComponentScan(basePackageClasses = PipelineTemplateModule.class)
 public class PipelineTemplateConfiguration {
+
+  @Bean
+  Handlebars handlebars() {
+    return new Handlebars();
+  }
+
+  @Bean
+  PipelineTemplateObjectMapper objectMapper() {
```

Comment:
name this method something other than 'objectMapper' (`pipelineTemplateObjectMapper` is probably a good bet - want to avoid conflicting with a standard object mapper in the context.